shard/src/ext/settings_cog.py

```diff
@@ -238,13 +254,44 @@ async def parse(self, ctx, msg, settings):
         return roles
 
 
+class GulagEmoji(SettingsElement):
+    def __init__(self):
+        super().__init__(
+            "Gulag Emoji",
+            PICK,
+            'This is the emoji that is used to tally up gulag votes. '
+            'React to this message to modify it:',
+            'gulag_emoji',
+            tags=["gulag"])
+
+    async def parse(self, ctx, msg, settings):
+        return str(msg)
```

Comment:
note: if you use a custom emoji from another guild you might break everything. I don't have nitro so i can't really test

Reply:
it just pastes the string in. I think it's pretty obvious what's going on so I'm going to merge it like this